Jenkinsfile

```diff
@@ -170,6 +170,11 @@ pipeline {
                             }
                         }
                     }
+                    post {
+                        success {
+                            archiveArtifacts artifacts: 'target/dependency-check-report.html'
```

Comment:
Is there any case that this file won't exist? Wondering if we should include the `allowEmptyArchive` option

Reply:
If the stage is successful the file has to exist because the maven goal we call creates the file. If the file does not exist the goal failed and the post block will never be called.